cli/src/cli.rs

```diff
@@ -38,6 +37,18 @@ pub struct ValidationWorkerCommand {
 	pub mem_id: String,
 }
 
+#[allow(missing_docs)]
+#[derive(Debug, StructOpt, Clone)]
+pub struct RunCmd {
+	#[allow(missing_docs)]
+	#[structopt(flatten)]
+	pub base: sc_cli::RunCmd,
+
+	/// Force using Kusama native runtime.
+	#[structopt(long = "force-kusama")]
+	pub force_kusama: bool,
```

Comment:
Do we still need this when we now can select the chain specs?

Reply:
Yes. It still applies when loading a spec from a json file.

Reply:
The native runtime is defined by the `Executor` and that is choosen by the `IsKusama` trait.

The loaded chain spec should return `is_kusama()`, than we don't need this.

Reply:
> The loaded chain spec should return is_kusama(), than we don't need this.

It does. The flag is to **override** this in case you want to ignore the id in the json file.

Reply:
Wasn't that a hack because we could not create a kusama genesis chain spec? That should be solved with this pr?

Reply:
I guess you are right we can remove it. If someone needs it for whatever reason they can just edit json file.